CaptureSDK/GiniCaptureSDK/Documentation/source/Customization guide.md

```diff
@@ -4,21 +4,55 @@ Customization guide
 The Gini Capture SDK components can be customized either through the `GiniConfiguration`, the `Localizable.string` file or through the assets. Here you can find a complete guide with the reference to every customizable item.
 
 - [Generic components](#generic-components)
+- [Onboarding screens](#onboarding-screens)
+- [No Authorized screen](#no-authorized-screen)
 - [Camera screen](#camera-screen)
+- [Gallery album screen](#gallery-album-screen)
 - [Review screen](#review-screen)
-- [Multipage Review screen](#multipage-review-screen)
 - [Analysis screen](#analysis-screen)
-- [Supported formats screen](#supported-formats-screen)
-- [Open with tutorial screen](#open-with-tutorial-screen)
-- [Capturing tips screen](#capturing-tips-screen)
-- [Gallery album screen](#gallery-album-screen)
-- [Onboarding screens](#onboarding-screens)
-- [Help screen](#help-screen)
+- [Help screens](#help-screens)
+- [No result screen](#no-result-screen)
+- [Error screens](#error-screens)
+
+## Colors
+
+We are providing a global color palette `GiniColors.xcassets` which you are free to override. The custom colors will be then applied on all screens.
+You can find the names of the colors [https://github.com/gini/gini-mobile-ios/tree/new-ui/CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Resources/GiniColors.xcassets](https://github.com/gini/gini-mobile-ios/tree/new-ui/CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Resources/GiniColors.xcassets).
```

Comment:
[Suggestion] maybe the text component or the link could be shortened, so it would look better for the viewer